App/HostedReactor.fs

```diff
@@ -0,0 +1,67 @@
+module HostedReactor
+
+open System
+open Microsoft.Extensions.Hosting
+open Propulsion.Internal
+open Propulsion.MessageDb
+
+type HostedReactor(connectionString, log, service: InvoiceNumberingReactor.Service, checkpointService: CheckpointStore.CheckpointService) =
+  inherit BackgroundService()
+
+  let stats =
+    { new Propulsion.Streams.Stats<_>(log, TimeSpan.FromMinutes 1, TimeSpan.FromMinutes 1) with
+        member _.HandleOk x = ()
+        member _.HandleExn(log, x) = () }
+
+  [<Literal>]
+  let groupName = "InvoiceNumberingReactor"
```

Comment:
PascalCase so it can be used as a constant pattern match and/or otherwise fit conventions